cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java

```diff
@@ -1130,6 +1136,91 @@ protected void handlePostAdminConfigurationAgent(HttpServletRequest request, Htt
         
     } // handlePostAdminConfigurationAgent
     
+    protected void handlePostAdminConfigurationInstance (HttpServletRequest request, HttpServletResponse response,
+            boolean v1) throws IOException {
+        response.setContentType("json;charset=utf-8");
```

Comment:
You're right. Perhaps I lost the "application/" in a copy paste. I'll change in all the file :)
Fixed in a01471d.